app/api/polar/webhooks/route.ts

```
@@ -0,0 +1,66 @@
+    if (signature !== expectedSignature) {
+      return c.json({ error: 'Invalid signature.' }, 400);
+    }
+
+    const event = JSON.parse(body);
+
+    console.log('Received Polar Webhook:', event.type);
+
+    // Handle different event types
+    switch (event.type) {
+      case 'customer.created':
+        const customerData = event.data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)